### DISCUSSION TITLE
Auto-shifting insights panels

### DISCUSSION THREAD

**Discussion by MattatDirectus** [directus/directus#16851] [Feature Requests]
When creating a new module, it would be cool to have the insight either fill in some of the negative space or "bump" other insights panels. 

Currently it overlays onto existing panels, causing a small issue/bug with it disappearing when trying to select. 
![Screenshot 2022-12-18 at 7 43 09 PM](https://user-images.githubusercontent.com/118303535/208329394-439202e8-5244-464f-a25d-5e9f14aff444.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
